README.md

```diff
@@ -44,6 +44,38 @@ In order to configure some aspects of layout, like the header, edit `config/tech
 
 If you move pages around and URLs change, make sure you set up redirects from the old URLs to the new URLs.
 
+## Making changes to a diagram
+Edit the draw.io files in `source/images/originals` folder by installing and using the draw.io desktop app.
+
+Use one drawio file per diagram.
+
+### Installing the draw.io desktop app
+Run the following commands to use the draw.io desktop app from the command line.
+
+```
+brew install --cask drawio
+alias draw.io='/Applications/draw.io.app/Contents/MacOS/draw.io'
+```
+
+### Editing and publishing a diagram
+Follow these steps:
+1. Create and modify a diagram using [draw.io](https://about.draw.io/).
+2. Save the draw.io file to the `source/images/originals` folder.
+3. Publish the diagram as a [scalable vector graphic (SVG)](https://www.w3.org/Graphics/SVG/).
+4. Save the SVG file to the `source/images` folder.
+ 
+### Useful commands
+Update a diagram:
+```
+draw.io source/images/originals/top-level-technical-diagram.drawio
+```
+Generate SVG versions of the diagrams and save them to the `source/images/originals` folder:
+```
+draw.io -x -o source/images/top-level-technical-diagram.svg source/images/originals/top-level-technical-diagram.drawio
+
+draw.io -x -o source/images/technical-flow-diagram.svg source/images/originals/technical-flow-diagram.drawio
+```
+
 ## Linting the GOV.UK One Login Technical Documentation
```

Comment:
Is "Technical Documentation" capitalised for a reason?

Reply:
@charleemg that's not a change that is a part of this pull request,
I think we should deal with that as part of a wider content review of the app.

That's not a line that @pauldougan has proposed a change for here.

Reply:
Going to mark this resolved, as it's out of scope for this PR,
though FYI @PippaClarkGDS we may have a wider content sweep a tech writer may be interested in if we've got work already in `main` we want to tidy up?